Sources/SwiftCompilerPluginMessageHandling/PluginMacroExpansionContext.swift

```diff
@@ -129,7 +129,7 @@ class SourceManager {
 
     var node = syntaxRegistry.get(source: syntaxInfo.source, kind: syntaxInfo.kind)
     if let operatorTable {
-      node = operatorTable.foldAll(node, errorHandler: { _ in /*ignore*/ })
+      node = operatorTable.foldAll(node, errorHandler: { _ in })  // ignore
```

Comment:
I think this is more readable with the block comment, same for the other `ignore` comments.

Reply:
I totally agree. I think in most cases if the comment block is placed between nodes, it shouldn’t be moved to the end?